database/txn/transaction.go

```diff
@@ -173,7 +173,9 @@ func (t *RetryingTxnRunner) StdTxn(ctx context.Context, db *sql.DB, fn func(cont
 	defer cancel()
 
 	return t.run(ctx, func(ctx context.Context) error {
-		tx, err := db.BeginTx(ctx, nil)
+		tx, err := db.BeginTx(ctx, &sql.TxOptions{
+			Isolation: sql.LevelReadCommitted,
+		})
```

Comment:
Drop this from this patch and propose it in another. We can talk about it then.

Reply:
I had a look into this, it doesn't do anything: See https://github.com/canonical/go-dqlite/blob/master/driver/driver.go#L483

Reply:
I'm very sorry, this is only a test I was doing (I wanted to check if it was possible to pass the `begin deferred` as a transaction type https://github.com/mattn/go-sqlite3/issues/400, not possible) so this has nothing to do here I just forgot to clean-up

Reply:
removed...